src/AzSK.ADO/Framework/Core/SVT/SVTControlAttestation.ps1

```diff
@@ -386,19 +432,25 @@ class SVTControlAttestation
 			#user provided justification text would be available only in bulk attestation mode.
 			if($null -ne $this.attestOptions -and  (-not [string]::IsNullOrWhiteSpace($this.attestOptions.JustificationText) -or $this.attestOptions.IsBulkClearModeOn))
 			{
-				$this.bulkAttestMode = $true;				
-					Write-Host "$([Constants]::SingleDashLine)" -ForegroundColor Yellow				
+				# if EnforceApprovedException is set to true in  control settings, the controls cannot be attested using bulk attestation mode
+				if ($this.enforceApprovedExceptionEnabled) {
+					write-host "The controls cannot be attested using bulk attest mode as approved exception is enforced in your org policy" -ForegroundColor Red
```

Comment:
Bulk attestation of controls is disabled for your project. - Red
Please attest controls on per-resource basis. - Yellow

Reply:
This will block bulk attestation for valid control ids (out of approved exception scope)

Reply:
changed as per comment